metadata-io/src/main/java/com/linkedin/metadata/params/ExtraIngestParamsHelper.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.metadata.params;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class ExtraIngestParamsHelper {
+
+    // condUpdate String format: "urn1+AspectName1=createdOn1;urn2+AspectName2=createdOn2;...;urnN+AspectNameN=createdOnN"
+
+    public static Map<String, Long> extractCondUpdate(String condUpdate) {
```

Comment:
As discussed, we should add a javadoc to explain the format of the `createdOn` parameters.